contracts/0.4.24/Lido.sol

```diff
@@ -908,17 +728,24 @@ contract Lido is ILido, StETH, AragonApp {
     }
 
     /**
-    * @dev Gets unaccounted (excess) Ether on this contract balance
-    */
+     * @dev Gets the amount of Ether temporary buffered on the StakingRouter contract balance
+     */
+    function _getStakingRouterBufferedEther() internal view returns (uint256) {
+        return STAKING_ROUTER_BUFFERED_ETHER_POSITION.getStorageUint256();
```

Comment:
Add defensive `assert(address(this).balance >= buffered);` or get rid of assert in `_getBufferedEther`

Reply:
remove assert from _getBufferedEther